tests/cases/access_test.py

```diff
@@ -89,6 +96,34 @@ def cookieHandler(self, **kwargs):
     def cookieForceHandler(self, **kwargs):
         return
 
+    @access.admin
+    def rawAdmin(self, **kwargs):
```

Comment:
This is redundant with ``adminHandler``. Same for ``userHandler``/``rawUser`` and ``publicHandler``/``rawPublic``.